### PR TITLE
fix: SQL Instance Pool - hard coding location of tests to make sure pipeline succeeds

### LINKED ISSUE #1744: [Failed pipeline] avm.res.sql.instance-pool
Failed run: https://github.com/Azure/bicep-registry-modules/actions/runs/8798441338

### PR DESCRIPTION
**Pull request by yashints**
## Description

Since we had to disable resource deletion on tests to false, we're hard coding the location to make sure consequent deployments pass.

Fixes #1744
Closes #1744

## Pipeline Reference

| Pipeline |
| -------- |
|     [avm.res.sql.instance-pool](https://github.com/yashints/bicep-registry-modules/actions/workflows/avm.res.sql.instance-pool.yml)     |

## Type of Change

- [ ] Update to CI Environment or utlities (Non-module effecting changes)
- [x] Azure Verified Module updates:
  - [ ] Bugfix containing backwards compatible bug fixes, and I have NOT bumped the MAJOR or MINOR version in `version.json`:
    - [ ] Someone has opened a bug report issue, and I have included "Closes #{bug_report_issue_number}" in the PR description.
    - [ ] The bug was found by the module author, and no one has opened an issue to report it yet.
  - [ ] Feature update backwards compatible feature updates, and I have bumped the MINOR version in `version.json`.
  - [ ] Breaking changes and I have bumped the MAJOR version in `version.json`.
  - [ ] Update to documentation

## Checklist

- [x] I'm sure there are no other open Pull Requests for the same update/change
- [x] I have run `Set-AVMModule` locally to generate the supporting module files.
- [x] My corresponding pipelines / checks run clean and green without any errors or warnings

### CODE REVIEW

**Comment by yashints** [comment]
cc: @eriqua requested fixes applied

File: .github/workflows/avm.res.sql.instance-pool.yml [hunk 1]
```diff
@@ -16,6 +16,9 @@ on:
         required: false
         default: true
       removeDeployment:
+        ### We had to set this to false because
```
Comment (eriqua) [inline .github/workflows/avm.res.sql.instance-pool.yml:19]: Please move this comment in one liner next to row 24

**Comment by eriqua** [comment]
> cc: @eriqua requested fixes applied

Thanks @yashints for jumping on this fix so quickly! 
Looks good, just a very minor comment

**Review by eriqua — APPROVED** [review]
🆗

**Pull request merged**